### ISSUE TITLE
Custom builds should not reference CDN-hosted versions of Dojo.

### ISSUE THREAD

**Issue by Leftium** [ivarconr/dojo-custom-build#1]
The [index-dev.html](https://github.com/ivarconr/dojo-custom-build/blob/0ca527e31a323d578597501f9173e660677e37ab/mywebapp/index-dev.html#L7) and [index-release.html](https://github.com/ivarconr/dojo-custom-build/blob/0ca527e31a323d578597501f9173e660677e37ab/mywebapp/index-release.html#L7) files use CDN-hosted versions of dojo.js. Custom builds must reference local (built) versions of dojo to have any effect.

Furthermore, changing the index files to point to the local dojo files breaks the application; the proper package configuration must be added to dojoConfig.

**Comment by ivarconr** [comment]
Yes, in most production systems you would generally host the dojo-core yourself. 

You don't have to though (running the project and inspect the network traffic, you will see that the custom layer is loaded from the custom build and not the CDN). In 1.7 you have to manually define if you want do build a custom version of the dojo-core. Dojo will not reload already loaded resources, and thus resources in the custom layer will not be loaded from the CDN in production mode. The nice thing about this approach is that: 
1. I do not have to provide my own CDN
2. CDN can host dojo-core: dojo.js
3. I host my custom layer (with dojo-components built in)
4. There is a safety net: if I screw up by excluding something in my custom layer it does not matter that much as dojo will use amd and load it directly from the CDN. 

But generally I agree with you that larger organization probably should host their on resources.
